code-of-conduct.md

```diff
@@ -1,13 +1,15 @@
 Codebridge is an open community founded on the principles of hospitality, transparency and mutual aid.
 
-We believe that we are able to use our collective interest in technology and social change in service of others and for the South African greater good.  We are committed to the values espoused in the [open source movement](https://opensource.org/osd-annotated) and [contributor covenant](https://www.contributor-covenant.org/) - not only in the projects that we champion, but also in the way that Codebridge itself is managed.
+We believe that we are able to use our collective interest in technology and social change in service of others and for the South African greater good.
+We are committed to the values espoused in the [Open Source Definition](https://opensource.org/osd-annotated) and [Contributor Covenant](https://www.contributor-covenant.org/)
+- not only in the projects that we champion, but also in the way that Codebridge itself is managed.
```

Comment:
Is this intentional?

Reply:
Yes; I reckon the separate lines make this paragraph a bit easier to read in text editors and in diffs.

Oh, but I see the last one actually gets misformatted! I'll fix that quick.

Reply:
Does the updated version look okay?